web_src/css/base.css

```diff
@@ -664,21 +721,12 @@ a.label,
   background: var(--color-hover);
 }
 
-.unselectable {
-  -webkit-touch-callout: none;
-  -webkit-user-select: none;
```

Comment:
What happened to this specific line?
That's no longer present…

Reply:
`-webkit-user-select` was removed because standard `user-select` is [well supported](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#browser_compatibility) in all browsers.

The selector itself was moved to line 321 and combined with others that had previoulsy used it via `extend` function  but we can move it back to this place again.

Reply:
I only meant the `-webkit-user-select`, the position is fairly unimportant for me (as long as we don't agree on some sort of order for selectors)

Reply:
I guess I can add it back so it's a clean refactor :)

Reply:
Restored it.